index.bs

```diff
@@ -11705,7 +11705,10 @@ provides basic, but reasonable results. It is used for the for the
 {{PannerNode}} when the {{PannerNode/panningModel}} attribute is set to
 "{{PanningModelType/equalpower}}", in which case the <em>elevation</em>
 value is ignored. This algorithm MUST be implemented using
-<a>a-rate</a> parameters.
+at the appropriate rate as specified by the
```

Comment:
This changes the spec, but I think we need to say this because the params can be either a-rate or k-rate.

Reply:
I agree, but let's confirm with @padenot.

Reply:
Aargh! I added a typo here that I need to fix:  "using at the appropriate..."

Reply:
Added @padenot as reviewer to look at this.